Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_42.md

```diff
@@ -0,0 +1,18 @@
+
+#### Playbooks
+
+##### Cortex ASM - ASM Alert
+
+Added support for patching using AWS Systems manger.
+
+##### Cortex ASM - AWS Enrichment
+
+Added the system ids to extract information from AWS Systems manger.
+
+##### Cortex ASM - Remediation Path Rules
+
+Added support for patching by AWS Systems manager data collection option.
+
+##### Cortex ASM - Remediation
+
+Added a remediation playbook for patching.
```

Comment:
```suggestion
Added the "AWS - Package Upgrade" remediation playbook for patching.
```

Reply:
Thank you for the suggestion, updated it